src/netius/base/poll.py

```
@@ -249,15 +249,30 @@ def poll(self):
     def is_edge(self):
         return True
 
+    def unsub_all(self, socket):
+        if not socket in self.write_o:
```

Ruff (0.16.2)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)